.editorconfig

```diff
@@ -0,0 +1,9 @@
+
+# top-most EditorConfig file
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+indent_style = tab
```

Comment:
This might make editors reformatting existing code. This depends on the used Editor/IDE and the local config resp. the used plugin.